docker-compose.yml

```diff
@@ -106,46 +111,50 @@ services:
       LAMBDA_FORWARD_URL: http://image-processor:8080
       DOCKER_HOST: unix:///var/run/docker.sock
     ports:
-      - "4567:4566"
+      - 4567:4566
 
 # Mock API gateway and image request handler
 # This will largely be replaced as we move forward and moved into use an LPA repo as it's mock.
 # Useful here for the time being.
-  api-gateway:
-    container_name: api-gateway
-    image: nginx:stable-alpine
-    depends_on:
-      - mock-image-request-handler
-    command: 'nginx -g "daemon off;"'
-    volumes:
-      - ./mock-services/image-request-handler/nginx.conf:/etc/nginx/conf.d/default.conf
-    ports:
-      - 7010:5000
-
-  mock-image-request-handler:
-    image: stoplight/prism:4
-    command: 'mock -h 0.0.0.0 /tmp/image-request-handler.yml'
-    volumes:
-      - ./docs/openapi/image-request-handler.yml:/tmp/image-request-handler.yml:ro
-    ports:
-      - 7011:4010
+#  api-gateway:
```

Comment:
not ready to remove yet as need for dev but don't want it coming up in docker-compose up -d